src/test/java/models/testGame.java

```diff
@@ -23,6 +23,13 @@ public void testGameBuildDeck(){
     }
 
     @Test
+    public void testSpanishGameBuildDect(){
+        Game g = new Game();
+        g.buildDeck('S');
+        assertEquals(40,g.deck.size());
```

Comment:
The build Deck Spanish and English could probably both be in the same test, just by creating two separate objects. This one test would ensure that our buildDeck method always works.